bluesky_widgets/_qt/tests/test_qt_search_input.py

```diff
@@ -0,0 +1,139 @@
+from datetime import datetime, timedelta
+import time
+import pytest
+from qtpy.QtCore import QDateTime
+from ...qt.search_input import QtSearchInput, ADA_LOVELACE_BIRTHDAY
+from ...components.search.search_input import SearchInput
+
+
+def as_qdatetime(datetime):
+    "Create QDateTime set as specified by datetime."
+    qdatetime = QDateTime()
+    qdatetime.setSecsSinceEpoch(datetime.timestamp())
+    return qdatetime
+
+
+def test_initial_state(qtbot):
+    "Check that 'All' is checked and since/until are set as expected."
```

Comment:
I'm getting errors when running pytest. 
`recursive dependency involving fixture 'qtbot' detected`
Don't we have to import the qtbot fixture from conftest?

Reply:
Interesting. What versions of pytest and pytest-qt are you running? It seems to work on Travis-CI and locally for me, so I suspect something is different about our environments.

> Don't we have to import the qtbot fixture from conftest?

That is a very reasonable question, but no, `pytest` is a little _too_ magical at times and one of the things it does is make everything in `conftest.py` automatically available as a test function parameter.

Reply:
Whoops...did not have pytest-qt installed. Now it's passing the tests.